x-pack/legacy/plugins/reporting/server/usage/types.d.ts

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-interface AvailableTotal {
+export interface AvailableTotal {
```

Comment:
This change doesn't seem to fix any problems, but I am not sure how this works without the export. Another file imports this interface!